packages/build/tests/plugins/snapshots/tests.js.md

```diff
@@ -2720,9 +2720,6 @@ Generated by [AVA](https://ava.li).
     > Context␊
       production␊
     ␊
-    > Loading plugins␊
```

Comment:
Those test snapshots changed because they trigger a build error based on the Node.js version. Since this happens earlier in the build flow, some build logs is not shown yet anymore when this error happens.